packages/flutter/test/cupertino/button_test.dart

```diff
@@ -125,4 +125,45 @@ void main() {
     // Still doesn't animate.
     expect(SchedulerBinding.instance.transientCallbackCount, equals(0));
   });
+
+  testWidgets('ActiveOpacity defaults to 0.1', (WidgetTester tester) async {
+    await tester.pumpWidget(new Center(child: new CupertinoButton(
+      child: new Text('Tap me'),
+      onPressed: () {},
```

Comment:
nit: here and below, empty code blocks should be `{ }` to distinguish them from empty Map literals which are `{}`.